apps/dashboard/src/components/project-navigation/style.scss

```diff
@@ -4,7 +4,6 @@
 	display: grid;
 	grid-template-columns: 1fr auto 1fr;
 	padding: 0 32px;
-	width: 100%;
```

Comment:
I believe this is actually quite important for the buttons to stay centered and not to move when you edit the title (and not be affected by the temporary buttons either).

Reply:
Fair enough, but we might need to find a better solution, maybe a flex-grow or something as it makes the Navigation container to overflow the viewport